packages/lib-classifier/src/components/Classifier/ClassifierContainer.spec.js

```diff
@@ -134,15 +134,15 @@ describe('components > ClassifierContainer', function () {
     it('should be able to view active workflows', async function () {
       await waitFor(() => {
         const subjectImage = screen.getByRole('img', {name: `Subject ${subjectSnapshot.id}` })
-        expect(subjectImage.getAttribute('href')).to.equal('https://static.zooniverse.org/www.zooniverse.org/assets/fe-project-subject-placeholder-800x600.png')
+        expect(subjectImage.getAttribute('href')).to.equal('https://foo.bar/example.png')
       })
       expect(workflowRequest.isDone()).to.be.true()
       expect(firstSubjectsRequest.isDone()).to.be.true()
       expect(secondSubjectsRequest.isDone()).to.be.false()
       expect(taskAnswers).to.have.lengthOf(workflowSnapshot.tasks.T0.answers.length)
       taskAnswers.forEach(radioButton => {
         expect(radioButton.name).to.equal('T0')
-        expect(radioButton.disabled).to.be.true()
+        expect(radioButton.disabled).to.be.false()
```

Comment:
Is the radio button no longer disabled in each test because the subject image is displayed rather than the placeholder by the time this line runs?

Reply:
Yeah. Tasks are disabled while the subject loads. Otherwise, you can classify without the image being present. 
https://github.com/zooniverse/Panoptes-Front-End/issues/5846

Reply:
Previously, these tests ran after the placeholder loaded but there’s no placeholder image any more.